### ISSUE TITLE
Files as well

### ISSUE THREAD

**Issue by mirojs** [aws-samples/bedrock-claude-chat#84]
Hello,

The sample looks nice!

Is there any chance of adding function like Creation, storage, and deletion of files to be used along with the internet info. in chat?

Thanks!

**Comment by statefb** [comment]
Hi mirojs,
Currently we are not going to develop that function. Do you have any good idea to implement it?

**Comment by javatask** [comment]
@statefb I would rephrase it by enable knowledge base as a RAG source for the chat

**Comment by statefb** [comment]
Implemented on #98
